cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -646,67 +771,19 @@ def main(self):
-        self.cmd_output_map['''{"caps": ["mon", "allow r, allow command quorum_status", "osd", "allow rwx pool=default.rgw.meta, allow r pool=.rgw.root, allow rw pool=default.rgw.control, allow x pool=default.rgw.buckets.index"], "entity": "client.healthchecker", "format": "json", "prefix": "auth get-or-create"}'''] = '''[{"entity":"client.healthchecker","key":"AQDFkbNeft5bFRAATndLNUSEKruozxiZi3lrdA==","caps":{"mon":"allow r, allow command quorum_status","osd":"allow rwx pool=default.rgw.meta, allow r pool=.rgw.root, allow rw pool=default.rgw.control, allow x pool=default.rgw.buckets.index"}}]'''
-        self.cmd_output_map['''{"caps": ["mon", "profile rbd", "osd", "profile rbd"], "entity": "client.csi-rbd-node", "format": "json", "prefix": "auth get-or-create"}'''] = '''[{"entity":"client.csi-rbd-node","key":"AQBOgrNeHbK1AxAAubYBeV8S1U/GPzq5SVeq6g==","caps":{"mon":"profile rbd","osd":"profile rbd"}}]'''
-        self.cmd_output_map['''{"caps": ["mon", "profile rbd", "mgr", "allow rw", "osd", "profile rbd"], "entity": "client.csi-rbd-provisioner", "format": "json", "prefix": "auth get-or-create"}'''] = '''[{"entity":"client.csi-rbd-provisioner","key":"AQBNgrNe1geyKxAA8ekViRdE+hss5OweYBkwNg==","caps":{"mgr":"allow rw","mon":"profile rbd","osd":"profile rbd"}}]'''
-        self.cmd_output_map['''{"caps": ["mon", "allow r", "mgr", "allow rw", "osd", "allow rw tag cephfs *=*", "mds", "allow rw"], "entity": "client.csi-cephfs-node", "format": "json", "prefix": "auth get-or-create"}'''] = '''[{"entity":"client.csi-cephfs-node","key":"AQBOgrNeENunKxAAPCmgE7R6G8DcXnaJ1F32qg==","caps":{"mds":"allow rw","mgr":"allow rw","mon":"allow r","osd":"allow rw tag cephfs *=*"}}]'''
-        self.cmd_output_map['''{"caps": ["mon", "allow r", "mgr", "allow rw", "osd", "allow rw tag cephfs metadata=*"], "entity": "client.csi-cephfs-provisioner", "format": "json", "prefix": "auth get-or-create"}'''] = '''[{"entity":"client.csi-cephfs-provisioner","key":"AQBOgrNeAFgcGBAAvGqKOAD0D3xxmVY0R912dg==","caps":{"mgr":"allow rw","mon":"allow r","osd":"allow rw tag cephfs metadata=*"}}]'''
-
-    def shutdown(self):
-        pass
-
-    def get_fsid(self):
-        return 'af4e1673-0b72-402d-990a-22d2919d0f1c'
-
-    def conf_read_file(self):
-        pass
-
-    def connect(self):
-        pass
-
-    def mon_command(self, cmd, out):
-        json_cmd = json.loads(cmd)
-        json_cmd_str = json.dumps(json_cmd, sort_keys=True)
-        cmd_output = self.cmd_output_map[json_cmd_str]
-        return self.return_val, \
-            cmd_output, \
-            "{}".format(self.err_message).encode('utf-8')
-
-    @classmethod
-    def Rados(conffile=None):
-        return DummyRados()
-
-
 # inorder to test the package,
 # cd <script_directory>
 # python -m unittest --verbose <script_name_without_dot_py>
 class TestRadosJSON(unittest.TestCase):
     def setUp(self):
-        print("{}".format("I am in setup"))
+        print("\nI am in setup")
```

Comment:
What is that supposed to mean?

Reply:
Previously it was like,
print("{}".format("I am in setup"))  which is equivalent to `go` s ==> fmt.Printf("%s", "I am in setup")
Since I have to print a constant string and not a variable, I could print it directly. Hence the change.

This line was to make sure each time the test-framework enters into a test method (`test_*` method), `setUp()` function is called as it's initialization process and `tearDown()` function is called once the tests are  done (either successfully or not).

Reply:
I guess what I meant is that "I am in setup" is difficult to understand.